new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/Domain.java

```diff
@@ -37,12 +37,52 @@ public static DomainList listDomainCustomResources(String namespace) {
     return Kubernetes.listDomains(namespace);
   }
 
+  /**
+   * Shut down a domain in the specified namespace.
+   * @param domainUid the domain to shut down
+   * @param namespace the namespace in which the domain exists
+   * @return true if patching domain custom resource succeeded, false otherwise
+   */
   public static boolean shutdown(String domainUid, String namespace) {
```

Comment:
modify the method name to shutdownDomain()

Reply:
having shutdown() is fine in implementation methods..In action class, we need the full method name shutdownDomain() as that's the method being called in test classes.

Reply:
fixed